nodejs/package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "test": "jest --no-cache",
     "lint": "eslint . --ext .js",
-    "build": "npx sequelize-cli db:migrate ./database.sqlite3"
+    "build": "npx sequelize-cli db:migrate --to ./database.sqlite3"
```

Comment:
For some reason, the original command does not work for me (The error is `Error: Error parsing url: ./database.sqlite3`). I've confirmed that `npx sequelize-cli db:migrate` is also fine.